plugins/usbdmx/EuroliteProFactory.cpp

```diff
@@ -46,12 +48,16 @@ const uint16_t EuroliteProFactory::VENDOR_ID_MK2 = 0x0403;
 
 const char EuroliteProFactory::ENABLE_EUROLITE_MK2_KEY[] =
     "enable_eurolite_mk2";
+const char EuroliteProFactory::EUROLITE_SERIALS_KEY[] =
+    "eurolite_serials";
 
 EuroliteProFactory::EuroliteProFactory(ola::usb::LibUsbAdaptor *adaptor,
                                        Preferences *preferences)
   : BaseWidgetFactory<class EurolitePro>("EuroliteProFactory"),
     m_adaptor(adaptor),
     m_enable_eurolite_mk2(IsEuroliteMk2Enabled(preferences)) {
+  StringSplit(preferences->GetValue(EUROLITE_SERIALS_KEY),
+              m_expected_eurolite_serials, ",");
```

Comment:
I'd probably go for multiple eurolite_serial lines, like this:
https://docs.openlighting.org/ola/conf/ola-nanoleaf.conf.html

As it's easier to manage each individual one or comment them out or whatever, and we don't actually care about the order of them (unlike we might elsewhere).

Then you'll need:
https://docs.openlighting.org/ola/doc/latest/classola_1_1_preferences.html#aa4bfa8a5218cfd23c573d23a52bdc88b